runtime/src/bank.rs

```diff
@@ -822,6 +856,15 @@ impl Bank {
 
         if *hash == Hash::default() {
             // finish up any deferred changes to account state
+
+            // DANGER: Adjust this guard after tds has transitioned to the eager rent
+            // collection and ABSOLUTELY after the mainnet-beta transitions to v1.1.
+            if !(self.operating_mode() == OperatingMode::Preview && self.epoch() < 47
+                || self.operating_mode() == OperatingMode::Stable
+                    && self.epoch() < Epoch::max_value())
```

Comment:
@mvines I've added a gate for mainnet-beta as well; time to merge and deploy devnet?: 5e5ae93860c68cbcb739bbf3d60a3c1230cec5d2